packages/studio-ui/src/App.tsx

```diff
@@ -3,6 +3,7 @@ import ActionsBar from "./components/ActionsBar";
 import Toast from "./components/Toast";
 import PreviewWithUseComponents from "./components/PreviewWithUseComponents";
 import LeftSidebar from "./components/LeftSidebar";
+import "react-tooltip/dist/react-tooltip.css";
```

Comment:
based on the [docs](https://react-tooltip.com/docs/getting-started#set-the-default-styling), I think we shouldn't need to manually import this CSS